dataflow/src/main/java/org/checkerframework/dataflow/cfg/JavaSource2CFG.java

```diff
@@ -0,0 +1,77 @@
+package org.checkerframework.dataflow.cfg;
+
+import com.sun.source.tree.CompilationUnitTree;
+import com.sun.tools.javac.file.JavacFileManager;
+import com.sun.tools.javac.main.JavaCompiler;
+import com.sun.tools.javac.util.Context;
+import com.sun.tools.javac.util.List;
+import com.sun.tools.javac.util.Options;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import javax.tools.JavaFileManager;
+import javax.tools.JavaFileObject;
+import org.checkerframework.dataflow.cfg.CFGProcessor.CFGProcessResult;
+
+/** Class to generate the DOT representation of the control flow graph of a given method. */
+public class JavaSource2CFG {
```

Comment:
Is it even worth splitting this into a separate class?
The only use is in the `CFGVisualizeLauncher` above, so these two could just be more instance methods there.
What motivated the renaming to leave out the DOT from the name?

Reply:
Thanks for the advice!

> What motivated the renaming to leave out the DOT from the name?

Renaming is because that we now both have `generateStringOfCFG` and `generateDOTofCFG`.

> Is it even worth splitting this into a separate class?

I totally agree and I have removed the class  `JavaSource2CFG`.